articles/machine-learning/service/how-to-train-ml-models.md

```diff
@@ -110,7 +111,8 @@ Parameter | Description | Default
 `custom_docker_image`| Name of the image you want to use. Only provide images available in public docker repositories (in this case Docker Hub). To use an image from a private docker repository, use the constructor's `environment_definition` parameter instead. [See example](https://github.com/Azure/MachineLearningNotebooks/blob/master/how-to-use-azureml/training-with-deep-learning/how-to-use-estimator/how-to-use-estimator.ipynb). | `None`
 `node_count`| Number of nodes to use for your training job. | `1`
 `process_count_per_node`| Number of processes (or "workers") to run on each node. In this case, you use the `2` GPUs available on each node.| `1`
-`distributed_backend`| Backend for launching distributed training, which the Estimator offers via MPI.  To carry out parallel or distributed training (e.g., `node_count`>1 or `process_count_per_node`>1 or both), set `distributed_backend='mpi'`. The MPI implementation used by AML is [Open MPI](https://www.open-mpi.org/).| `None`
+
+`distributed_training`| [MPIConfiguration ]('https://docs.microsoft.com/python/api/azureml-core/azureml.core.runconfig.mpiconfiguration?view=azure-ml-py') object for launching distributed training using MPI backend. The MPI implementation used by Azure Machine Learning is [Open MPI]('https://www.open-mpi.org/').  | `None`
```

Comment:
Currently we use IntelMPI as default MPI because only IntelMPI works with Azure InfiniBand. We will change to OpenMpi in the future when SR-IOV is available in all regions but this change won't come soon.

Reply:
Do you think it make sense to omit the mention of MPI implementation all along for now until we change to OpenMpi?

Reply:
Sounds good to me.

Reply:
@bninghu  I have made the change, please review

Reply:
I still see OpenMpi being mentioned?

Reply:
it should be on latest commit